stock_picking_invoice_link/models/account_move.py

```diff
@@ -56,6 +61,13 @@ class AccountMoveLine(models.Model):
         column2="move_id",
         string="Related Stock Moves",
         readonly=True,
-        help="Related stock moves "
-        "(only when the invoice has been generated from a sale order).",
+        copy=False,
+        help="Related stock moves (only when the invoice has been "
+        "generated from a sale order).",
     )
+
+    # pylint: disable=W8110
+    def _copy_data_extend_business_fields(self, values):
+        # OVERRIDE to copy the 'move_line_ids' field as well.
+        super()._copy_data_extend_business_fields(values)
+        values["move_line_ids"] = [(6, None, self.move_line_ids.ids)]
```

Comment:
Could you explain why overriding this ?

Reply:
Credit note lines wouldn't be linked to stock moves, otherwise. IMO, this part itself is valid.

However the the changes proposed in #750 should also be applied or stock moves get mistakenly regarded as consumed and the subsequent invoice won't be linked to the picking when a full refund is done with credit note (i.e. invoiced quantity of the sales order line is deducted). Maybe this part can be done later as #750 gets processed.

Reply:
This is not correct, as duplicating an invoice will also copy the linked moves. This is covered already in previous versions with latest changes like #1074

Reply:
> as duplicating an invoice will also copy the linked moves

I don't think this is true as it requires 'include_business_fields' context for this part of the values to be copied, which is not the case with simple duplication. If the changes from #1074 takes care of the links nicely then this part can be argued to be removed as necessary in the port process.